webapp/src/main/java/org/opentestsystem/rdw/admin/service/CsvValidationService.java

```diff
@@ -0,0 +1,61 @@
+package org.opentestsystem.rdw.admin.service;
+
+import com.google.common.collect.ImmutableSet;
+import org.apache.commons.csv.CSVRecord;
+import org.opentestsystem.rdw.admin.common.security.PermissionScope;
+import org.opentestsystem.rdw.admin.model.CsvValidationFailure;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Implementations of this interface are responsible for validating
+ * a CSV representation of a Student Group submission.
+ */
+public interface CsvValidationService {
+
+    final static String HeaderSchool = "school_natural_id";
+    final static String HeaderSubject = "subject_code";
+    final static String HeaderGroup = "group_name";
+    final static String HeaderUser = "group_user_login";
+    final static String HeaderYear = "school_year";
+    final static String HeaderSSID = "student_ssid";
+
+    final static Set<String> RequiredHeaders = ImmutableSet.of(
+            HeaderSchool,
+            HeaderSubject,
+            HeaderGroup,
+            HeaderUser,
+            HeaderYear,
+            HeaderSSID
+    );
```

Comment:
I was resistant as well.  But apparently the industry like constants on Interfaces rather than implementations.  I could break them into a separate constant class, I suppose...  CsvGroupHeaders.java?